tests/fatrace-currentmount

```diff
@@ -1,79 +0,0 @@
-#!/bin/sh
-set -euC
-
-LOG="$AUTOPKGTEST_TMP/fatrace.log"
-echo "starting fatrace..."
-fatrace --current-mount -s 2 -o $LOG &
-sleep 1
-
-echo "read a file..."
-head NEWS > /dev/null
-
-echo "create a file..."
-TEST_FILE=test.txt
-touch "$TEST_FILE"
-set +C
-bash -c "echo hello > '$TEST_FILE'"
-set -C
-rm "$TEST_FILE"
-
-echo "moving a file within the same directory"
-touch "$TEST_FILE"
-mv "$TEST_FILE" "${TEST_FILE}.2"
-mkdir dest
-mv "${TEST_FILE}.2" dest/"${TEST_FILE}.2"
-rm dest/"${TEST_FILE}.2"
-rmdir dest
-
-echo "robustness against ELOOP"
-ln -s nothing link
-python3 -c 'import os; os.open("link", os.RDONLY | os.NOFOLLOW)' 2>/dev/null || true
```

Comment:
Dropped this line. I ported it first, but it's nonsensical, it doesn't actually check anything.